aether-producer/aether/producer/kernel_api.py

```diff
@@ -44,11 +44,20 @@
     '&page_size={page_size}'
     '&fields=id,schema,schema_name,schema_definition'
 )
-_ENTITIES_URL = (
+_ENTITIES_SINGLE_URL = (
     f'{_KERNEL_URL}/'
     'entities.json?'
     '&page_size={page_size}'
-    '&fields=id,modified,payload'
+    '&fields=id,modified,payload,schema'
+    '&ordering=modified'
+    '&modified__gt={modified}'
+    '&schema={schema}'
+)
+_ENTITIES_ALL_URL = (
+    f'{_KERNEL_URL}/'
+    'entities.json?'
+    '&page_size={page_size}'
+    '&fields=id,modified,payload,schema'
```

Comment:
As far as I see `_ENTITIES_SINGLE_URL` and `_ENTITIES_ALL_URL` are exactly the same :shrug: 

![image](https://user-images.githubusercontent.com/8459537/86146067-0367a000-baf8-11ea-8bc7-7f8c22b62c82.png)